pages/news/2020-XX-XX-new-in-2-3-serato-support.html

```diff
@@ -0,0 +1,78 @@
+It took a months of work, but we finally manage to [reverse-engineer the binary formats](https://github.com/Holzhaus/serato-tags) that are used by Serato to a degree that allows us to parse this information and use it in Mixxx.
+
+## Importing your Hotcues
+
+This means you can import the positions, labels and colors of your hotcues into Mixxx automatically when you load the track for the first time.
+
+{# TODO: Insert side-by-side screenshot of hotcues in Mixxx and Serato once the skins are finalized. #}
+
+If you already added a track to your Mixxx library before support for reading Serato's hotcues was added, you can trigger a reimport of the metadata via the track context menu.
+Note that this will clear your existing cuepoints in Mixxx if the track has any Serato hotcues.
+
+{# TODO: Add screenshot of reimport metadata context menu entry. #}
+
+Unfortunately, a myriad of encoders and decoders for lossy formats such as MP3 and M4A/AAC exist, which leads to a situation where all decodes detect slightly different track start and end times for files from different sources.
+Hence, your cues might end up shifted by a few milliseconds.
+
+We've done our best to mitigate the problem, but if you experience issues we'd appreciate if you get in touch with us on Zulip and work with us to make the offset correction more accurate.
+Additionally, we've added a way to shift all cues for a track at once. This makes it possible to fix the cue positions if Mixxx fails to determine the correct offsets.
+
+## Support for other Serato metadata
+
+Mixxx 2.3 will also import saved loops from Serato, but proper support for saving and restoring loops is scheduled for 2.4.
+For the time being, we just import these loops into the Mixxx database and allows using their start position as regular hotcues.
+That way, you can start using your loops once 2.4 is out without the need to reimport your metadata (which would undo your cue modifications in Mixxx).
+
+Other Serato metadata that we parse include the track color and the BPM lock status.
+Serato also stores other information like the overview waveform image and the beatgrid in the file tags, but we don't import that information in Mixxx 2.3.
+[Serato Flip](https://serato.com/dj/pro/expansions/flip) hotcue macros are not imported either, because Mixxx does not support that kind of functionality yet.
+This is going to change with this year's GSoC project by Janek Fischer, so hopefully we'll be able to be able to import them in the future.
+
+## Writing Serato's Tags?
+
+Currently, it's not possible to write cues from Mixxx into Serato metadata tags.
+
+At Mixxx, we believe vendor lock-in is a bad thing and that users should be free to use the DJ software they like best.
+While these changes allow your to migrate from Serato to Mixxx with less friction we don't want to lock user into Mixxx either.
+
+This means were looking to adding support for Serato metadata export in upcoming Mixxx releases.
```

Comment:
s/were looking to adding/we're looking to add/

Reply:
You can now suggest changes directly via GitHub so you don't have to use sed syntax. Click this icon:
![image](https://user-images.githubusercontent.com/9455094/81294702-deddd300-9034-11ea-905c-c3f8c5c25a3f.png)